server/apps/validate/validate.py

```diff
@@ -38,10 +37,22 @@ def create(self, docs, **kwargs):
 
         return [doc['errors'] for doc in docs]
 
+    def _get_validators(self, action, item_type, profile=None):
+        """Get validators.
+
+        In case there is profile defined for item with respective content type it will
+        use its schema for validations, otherwise it will fall back to action/item_type filter.
+        """
```

Comment:
I would recommend adding parameter descriptions as well, it will make it easier to drill-down through the code without an unnecessary additional guessing burden (e.g. when debugging).